app/assets/images/payment_icons/i_pay_secure_payments.svg

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<svg viewBox="211.13499450683594 193 38 24" width="525" height="169.00006103515625" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Isn't the requested width="38" height="24"

Reply:
Noted thank you. Will advise designers to amend the issue.